app/javascript/components/Product/CardGrid.tsx

```diff
@@ -351,7 +351,7 @@ export const CardGrid = ({
       ) : (
         <div>
           <div className="product-card-grid" ref={gridRef}>
-            {results?.products.map((result) => <Card key={result.permalink} product={result} />) ??
+            {results?.products.map((result, idx) => <Card key={result.permalink} product={result} eager={idx < 3} />) ??
```

Comment:
I think we might need to allow for more than this by default, the discover page shows the first 4 products in the grid on my screen